src/docfx/lib/TemplateEngineUtility.cs

```diff
@@ -0,0 +1,66 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Generic;
+using Newtonsoft.Json.Linq;
+
+namespace Microsoft.Docs.Build
+{
+    internal static class TemplateEngineUtility
+    {
+        private static readonly HashSet<string> s_outputAbsoluteUrlYamlMime = new(StringComparer.OrdinalIgnoreCase)
+        {
+            "Architecture",
+            "TSType",
+            "TSEnum",
+        };
+
+        private static readonly HashSet<string> s_yamlMimesMigratedFromMarkdown = new(StringComparer.OrdinalIgnoreCase)
+        {
+            "Architecture",
+            "Hub",
+            "Landing",
+            "LandingData",
+        };
+
+        public static bool OutputAbsoluteUrl(string? mime) => mime != null && s_outputAbsoluteUrlYamlMime.Contains(mime);
+
+        public static bool IsConceptual(string? mime) => "Conceptual".Equals(mime, StringComparison.OrdinalIgnoreCase);
+
+        public static bool IsLandingData(string? mime) => "LandingData".Equals(mime, StringComparison.OrdinalIgnoreCase);
+
+        public static bool IsMigratedFromMarkdown(string? mime)
```

Comment:
These mime type related static method feels more related to `JsonSchemaProvider`

Reply:
Fixed.